src/DssSpell.t.sol

```diff
@@ -282,7 +282,7 @@ contract DssSpellTest is DssSpellTestBase {
     }
 
     // leave public for now as this is acting like a config tests
-    function testPSMs() public {
+    function testPSMs() private { // Disabled since some PSMs on mainnet are maxed out
```

Comment:
To what extent are we flying blind when we turn this off? Is anything checked by the config file or are *all* the tests disables (besides the one you have for `testPSMTinTout` below) when we private this?

Reply:
This test had bugs and was temporarily replaced in the mainnet spell.
Fix is in a separate [PR](https://github.com/makerdao/spells-mainnet/pull/326) and ideally would be merged soon.